.github/composite/exp1/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Run experiment 1
+description: "Run Gradle Experiment 1"
+
+inputs:
+  repositoryUrl:
+    description: "Project repository URL"
+    required: true
+  branch:
+    description: "Git Branch"
+    required: true
+  task:
+    description: "Gradle task"
+    required: true
+  projectDir:
+    description: "Project directory"
+    required: false
+    default: .
+  gradleEnterpriseUrl:
+    description: "Gradle Enterprise URL"
+    required: true
+outputs:
+  buildScanId1:
+    description: "First build scan identifier"
+    value: ${{ steps.run.outputs.buildScanId1 }}
+  buildScanId2:
+    description: "Second build scan identifier"
+    value: ${{ steps.run.outputs.buildScanId2 }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Run Gradle Experiment 1
+      id: run
+      run: |
+        cd gradle-enterprise-gradle-build-validation
+
+        # add instructions to save build scan urls
+        BUILD_SCAN_FILE_1=$(pwd)/exp1-build-scan-url-1.txt
+        BUILD_SCAN_FILE_2=$(pwd)/exp1-build-scan-url-2.txt
+        echo "echo \${build_scan_urls[0]} > ${BUILD_SCAN_FILE_1}" >> 01-validate-incremental-building.sh
+        echo "echo \${build_scan_urls[1]} > ${BUILD_SCAN_FILE_2}" >> 01-validate-incremental-building.sh
+
```

Comment:
This can be simplified since running the experiment will create files that contain the build scan ids already.

Reply:
We should consider adding the entire experiment receipt created by the validation scripts as an asset/resource attached to the GH workflow execution.

Reply:
> This can be simplified since running the experiment will create files that contain the build scan ids already.

This is not so easy to retrieve the receipt as it is part of a randomly named generated folder.
In addition to that, even if unlikely, you could think of a case where several runs could happen for an experiment, forcing to sort the generated folders by timestamp.
Having a way to set as an argument the absolute path to the receipt could be another way to get the job done.

> We should consider adding the entire experiment receipt created by the validation scripts as an asset/resource attached to the GH workflow execution.

The only thing we need from the receipt is actually the build scan id to query the REST API later on

Reply:
It's possible to get the experiment directory from the summary printed at the end of the experiment. In that directory is a the receipt, but there is also a file that contains just the build scan URLs.

Reply:
I'd like not to parse the summary, looking for Build scan first build / Build scan second build sounds fragile. 
Looking into the `csv` might lead to some unexpected refactoring implication at some point imo.

Having a way to set the experiment folder sounds more solid from my perspective.

Reply:
Now that we have the symlink to the latest experiment, we can get the scan URLs from the csv. (once we release 1.0.2)